app/models/errors.rb

```diff
@@ -48,4 +48,10 @@ def message
       "Storage location kit doesn't match"
     end
   end
+
+  class StorageLocationNotEmpty < StandardError
+    def message
+      "Cannot deactivate storage location containing inventory items with non-zero quantities"
+    end
+  end
```

Comment:
Mentioned below in another file, that we could handle this in the server object without managing another fairly specific error.

I think the `model/errors.rb` is a good idea for errors that we'll use multiple times. But in this case, it is very specific.